okcurl/build.gradle

```diff
@@ -35,3 +46,21 @@ dependencies {
 shadowJar {
   mergeServiceFiles()
 }
+
+graal {
+  mainClass "okhttp3.curl.Main"
+  outputName "okcurl"
+  graalVersion "20.2.0"
+  javaVersion "11"
+
+  option "--enable-https"
```

Comment:
this is a graal option, not an OkHttp option?!

Reply:
It makes sense, do you want a keystore and native crypto libraries bundled?

Reply:
Yep!